documentation/src/main/java/com/vaadin/flow/tutorial/routing/RouterNavigation.java

```diff
@@ -31,8 +34,16 @@ void navigation() {
         NativeButton button = new NativeButton(
                 "Navigate to company");
         button.addClickListener(e ->
-            button.getUI().ifPresent(ui ->
-                    ui.navigate("company"))
+                button.getUI().ifPresent(ui ->
+                        ui.navigate("company"))
+        );
+
+        NativeButton editButton = new NativeButton(
+                "Edit user details");
+        editButton.addClickListener(e ->
+                button.getUI().ifPresent(ui -> ui.navigate(
```

Comment:
```suggestion
                editButton.getUI().ifPresent(ui -> ui.navigate(
```

Reply:
Needed update also in [tutorial-routing-navigation.asciidoc](https://github.com/vaadin/flow-and-components-documentation/blob/master/documentation/routing/tutorial-routing-navigation.asciidoc)